package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "dashboards-desktop",
+  "version": "1.0.0",
+  "description": "<img src=\"https://opensearch.org/assets/brand/SVG/Logo/opensearch_dashboards_logo_darkmode.svg\" height=\"64px\"/>",
+  "main": "src/main.js",
+  "scripts": {
+    "start": "electron ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/johnnyon-amzn/dashboards-desktop-poc.git"
```

Comment:
Is it possible not have personal information "johnnyon-amzn"? I thought that should point to "opensearch-project/dashboards-desktop", the same as other url.

Reply:
I updated in commit a1eb3eb to reference main github page